test/e2e/utils_test.go

```diff
@@ -0,0 +1,21 @@
+package e2e
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+)
+
+// PowerShellCommand is a wrapper to run something in powershell.  It takes a command and args.
+// And it returns stdout as a string
+func PowerShellCommand(commandName string, args []string) (string, error) {
```

Comment:
Is this still used?

Reply:
No, but a the test suite gets filled out, I'm certain it will be.